tiatoolbox/models/architecture/hovernet.py

```diff
@@ -716,16 +754,21 @@ def infer_batch(model, batch_data, on_gpu):
         aggregation.
 
         Args:
-            model (nn.Module): PyTorch defined model.
-            batch_data (ndarray): a batch of data generated by
-                torch.utils.data.DataLoader.
-            on_gpu (bool): Whether to run inference on a GPU.
+            model (nn.Module):
+                PyTorch defined model.
+            batch_data (ndarray):
+                A batch of data generated by
+                `torch.utils.data.DataLoader`.
+            on_gpu (bool):
+                Whether to run inference on a GPU.
 
         Returns:
-            List of output from each head, each head is expected to contain
-            N predictions for N input patches. There are two cases, one
-            with 2 heads (Nuclei Pixels `np` and Hover `hv`) or with 2 heads
-            (`np`, `hv`, and Nuclei Types `tp`).
+            tuple:
+                Tuple of output from each head, each head is expected to
```

Comment:
New sentence after 'Tuple of output from each head'. Also remove 'Tuple' at beginning. This is already defined as the output dtype